master/coords.html

```diff
@@ -834,15 +834,15 @@ <h2 id="BoundingBoxes">Bounding boxes</h2>
 <p>Three kinds of bounding boxes can be computed for an element:</p>
 
 <ol>
-  <li>The <dfn id="TermObjectBoundingBox" data-dfn-type="dfn" data-export="">object bounding box</dfn> is the bounding box that contains only
+  <li>The <dfn id="TermObjectBoundingBox" data-dfn-type="dfn" data-dfn-export>object bounding box</dfn> is the bounding box that contains only
   an element's geometric shape.  For <a>basic shapes</a>, this is the area
   that is filled.  Unless otherwise specified, this is what is meant by the
   unqualified term "bounding box".</li>
 
-  <li>The <dfn id="TermStrokeBoundingBox" data-dfn-type="dfn" data-export="">stroke bounding box</dfn> is the bounding box that contains
+  <li>The <dfn id="TermStrokeBoundingBox" data-dfn-type="dfn" data-dfn-export>stroke bounding box</dfn> is the bounding box that contains
   an element's geometric shape and its <a>stroke shape</a>.</li>
 
-  <li>The <dfn id="TermDecoratedBoundingBox" data-dfn-type="dfn" data-export="">decorated bounding box</dfn> is the bounding box that contains
```

Comment:
Was the `data-export` set manually? On that note... our XML based tools are not affected by attributes w/o values?

Reply:
I'm not sure but it was in a bunch of places. I'm not familiar with the XML tools, but there was a definitions class all over the source in single quotes. Wouldn't that break the XML tools too?